### PR TITLE
Show update stack widget on mismatched template/stack revisions

### PR DESCRIPTION
**Pull request by usirin**
This has one issue that unread count update comes after from stack template update and there is no way to wait for that event to happen without listening changes on `computeController`

### CODE REVIEW

File: client/app/lib/remote-extensions/stacktemplate.coffee [hunk 1]
```diff
@@ -6,6 +6,9 @@ remote = require('../remote')
 module.exports = class JStackTemplate extends remote.api.JStackTemplate
 
 
+  getRevisionStatus: -> @template.sum
```
Comment (gokmen) [inline client/app/lib/remote-extensions/stacktemplate.coffee]: `@getAt 'template.sum'`

File: client/app/lib/sidebarcontroller.coffee [hunk 2]
```diff
@@ -60,6 +62,29 @@ module.exports = class SidebarController extends kd.Controller
...
+      computeController.storage.on 'change:templates', (event) =>
+        { operation, value: template } = event
+
+        # add a handler for each template push
+        if operation is 'push' and not @_templateEventCache[template.getId()]
+          debug 'binding template events', template
+          template.on 'update', =>
+            debug 'template has an update event', template
+            stacks = computeController.storage.stacks.get().filter (stack) =>
+              stack.baseStackId is template.getId()
```
Comment (gokmen) [inline client/app/lib/sidebarcontroller.coffee]: `stacks = computeController.storage.stacks.query 'baseStackId', template.getId()`

**Review by gokmen — APPROVED** [review]
👍 just minors

**Comment by gokmen** [comment]
```
$(npm bin)/coffeelint --quiet .
  ✗ client/app/lib/sidebarcontroller.coffee
     ✗ #73: Unnecessary fat arrow.

✗ Lint! » 1 error and 0 warnings in 1258 files
```

we can merge after coffeelint issue fixed.

**Comment by usirin** [comment]
This pr results in updated stack event popping up each time name of the template is changing for the owner. I will dig this more, adding `wip` tag.

**Comment by usirin** [comment]
@gokmen i started not getting events for `_revisionStatus` changes on `JComputeStack` therefore the unread count icon doesn't appear on realtime events, any ideas?

btw, title change event problem i mentioned above is not happening with `nse` 🤘

**Comment by gokmen** [comment]
@usirin that might be related with pubnub, it's happening sometimes for the development config which I'm ending up with restarting everything or checking out the pubnub status.

**Pull request closed without merging**